lib/web/apiserver_test.go

```diff
@@ -1952,11 +1949,9 @@ func (f byTimeAndIndex) Swap(i, j int) {
 // TestSearchClusterEvents makes sure web API allows querying events by type.
 func TestSearchClusterEvents(t *testing.T) {
 	t.Parallel()
-	// We need a clock that uses the current time here to work around
-	// the fact that filelog doesn't support emitting past events.
```

Comment:
> 	// We need a clock that uses the current time here to work around
>	// the fact that filelog doesn't support emitting past events.

This assumption seems to be incorrect. From what I can tell, `filelog` has no issue writing past events, assuming they are emitted in order.